### PR TITLE
fixing #15396 & #15397

### LINKED ISSUE #15396: Accessibility of userlist table should be improved
The current implementation of the `userlist` table (templates/users/part.userlist.php) misses `scope` attributes on the `th` elements. Moreover there should be one `th` per row. IMHO it should be the name element.

Further information at: http://webaim.org/techniques/tables/data

### LINKED ISSUE #15397: Markup Cleanup of userlist table
The markup should be cleaned and have unified identation, quotation marks and so on.

### PR DESCRIPTION
**Pull request by Volker-E**
fixes #15396
fixes #15397

### CODE REVIEW

**Comment by MorrisJobke** [comment]
@owncloud-bot this is okay to test

**Comment by MorrisJobke** [comment]
Works :+1:

**Comment by MorrisJobke** [comment]
@owncloud-bot retest this please

**Comment by MorrisJobke** [comment]
Build failures are unrelated

**Comment by MorrisJobke** [comment]
@owncloud-bot retest this please

**Comment by MorrisJobke** [comment]
@owncloud/designers Have a look at this ;)

cc @LukasReschke ;)

**Comment by MorrisJobke** [comment]
@DeepDiver1975 Something for 8.1 or for 8.2?

File: settings/templates/users/part.userlist.php [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
 		<!-- the following <tr> is used as a template for the JS part -->
 		<tr style="display:none">
-			<?php if ($_['enableAvatars']): ?>
-				<td class="avatar"><div class="avatardiv"></div></td>
-			<?php endif; ?>
-			<td class="name"></td>
+		<?php if ($_['enableAvatars']): ?>
+			<td class="avatar"><div class="avatardiv"></div></td>
+		<?php endif; ?>
+			<th class="name" scope="row"></th>
```
Comment (jancborchardt) [inline settings/templates/users/part.userlist.php]: This should stay a `td` though, right?

Comment (Volker-E) [inline settings/templates/users/part.userlist.php]: @jancborchardt For screen-readers a `th` is **huge** improvement here. See http://webaim.org/techniques/tables/data

**Comment by DeepDiver1975** [comment]
> @DeepDiver1975 Something for 8.1 or for 8.2?

I'm fine to merge this into 8.1 - @jancborchardt are you okay with these changes? THX

File: settings/templates/users/part.userlist.php (hunk 1 above)
Comment (jancborchardt) [inline settings/templates/users/part.userlist.php]: Aahh sure, I get it now. Thank you! :)

**Comment by jancborchardt** [comment]
:+1:

**Pull request merged**